CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing to Eclipse uProtocol
```

Comment:
wasn't the idea to link to https://github.com/eclipse-uprotocol/.github/blob/main/CONTRIBUTING.adoc instead of re-writing this?

Reply:
Yep you're right.